style.css

```diff
@@ -4,20 +4,37 @@
   font-family: "Poppins", sans-serif;
 }
 
+body {
+  margin: 0 !important;
+}
+
```

Comment:
- Hey, according to the microverse [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), you should not overuse the `important` keyword, while you've used it once, please, consider you shouldn't use it at all except you're using a CSS framework. please, kindly try and fix this by not using the keyword in your styles. let's improve on this please :pray: 

- Here, you can add this reset to fix the issue below:

```
*,
*::before,
*::after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```

if you've explicitly used the box-sizing, you can remove it.

Reply:
Solved all required changes! Thanks for review